integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -0,0 +1,219 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyWlsRemoteConsole;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.setPodAntiAffinity;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.shutdownWlsRemoteConsole;
+import static oracle.weblogic.kubernetes.utils.TestUtils.callWebAppAndWaitTillReady;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("Test WebLogic remote console connecting to mii domain")
+@IntegrationTest
+class ItRemoteConsole {
+
+  private static String domainNamespace = null;
+
+  // domain constants
+  private static final String domainUid = "domain1";
+  private static final String clusterName = "cluster-1";
+  private static final int replicaCount = 1;
+  private static final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain();
+
+  }
+
+  /**
+   * Verify WLS Remote Console installation is successful.
+   * Verify k8s WebLogic domain is accessible through remote console.
+   */
+  @Test
+  @DisplayName("Verify WLS Remote Console installation is successful")
+  public void testWlsRemoteConsoleInstallation() {
+
+    assertTrue(installAndVerifyWlsRemoteConsole(), "Remote Console installation failed");
+
+    int nodePort = getServiceNodePort(
+        domainNamespace, getExternalServicePodName(adminServerPodName), "default");
+    assertTrue(nodePort != -1,
+        "Could not get the default external service node port");
+    logger.info("Found the default service nodePort {0}", nodePort);
+    logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
+    String curlCmd1 = "curl -s -L -v --user weblogic:welcome1 --show-error --noproxy '*' "
```

Comment:
modify the variable name to curlCmd

Reply:
Now the command is like: now the curl command is: curl -v --user weblogic:welcome1 -H Content-Type:application/json -d “{ \“domainUrl\“: \“http://100.111.141.83:30929\” }” http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null
And return HTTP status code 201, which is meaningful.